2-patterns_integrations/src/test/java/com/epam/dhontar/aqamp/BaseTest.java

```diff
@@ -0,0 +1,99 @@
+package com.epam.dhontar.aqamp;
+
+import static com.epam.dhontar.aqamp.utils.enums.Constants.TESTRAIL_PWD;
+import static com.epam.dhontar.aqamp.utils.enums.Constants.TESTRAIL_URL;
+import static com.epam.dhontar.aqamp.utils.enums.Constants.TESTRAIL_USER;
+
+import static java.lang.String.format;
+
+import com.epam.dhontar.aqamp.observer.Chat;
+import com.epam.dhontar.aqamp.observer.ChatSubscriber;
+import com.epam.dhontar.aqamp.utils.integrations.slack.SlackClient;
+import com.epam.dhontar.aqamp.utils.integrations.testrail.TestRails;
+import com.epam.dhontar.aqamp.utils.integrations.testrail.bindings.APIClient;
+import com.epam.dhontar.aqamp.utils.integrations.testrail.bindings.APIException;
+import org.json.simple.JSONObject;
+import org.testng.ITestContext;
+import org.testng.ITestResult;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public abstract class BaseTest {
+    private static final String USER_DIR = System.getProperty("user.dir");
+    private static final Path PATH = Paths.get(USER_DIR);
+    private static final String PROJECT_NAME = PATH.getFileName().toString();
```

Comment:
Don't complicate please. Remove this functionality and just add "LocalTime", for example instead of it.

Reply:
Just wanted to see pretty test run name. Simplified